cgi-bin/DW/Console/Command/ResetPBEMToken.pm

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/perl
+#
+# DW::Console::Command::ResetPBEMToken
+#
+# Console command for resetting PBEM tokens
+#
+# Authors:
+#      Adam Bernard <user name="pseudomonas" site="dreamwidth.org"">
+#
+# Copyright (c) 2015 by Dreamwidth Studios, LLC.
+#
+# This program is free software; you may redistribute it and/or modify it under
+# the same terms as Perl itself.  For a copy of the license, please reference
+# 'perldoc perlartistic' or 'perldoc perlgpl'.
+
+package DW::Console::Command::ResetPBEMToken;
+use strict;
+
+use base qw/ LJ::Console::Command /;
+
+sub cmd { 'reset_token' }
+sub desc { 'Reset post-by-email token.' }
+sub args_desc {
+    [
+        'username' => 'Username of the account whose token is to be reset',
+        'reason' => 'Reason for resetting it.',
+    ]
+}
+sub usage { '<username> <reason>' }
+
+sub can_execute {
+    my $remote = LJ::get_remote();
+    return $remote && $remote->has_priv( "reset_email" );
+}
+
+sub execute {
+    my ($self, $username, $reason, @args) = @_;
+
+    return $self->error("This command takes two arguments. Consult the reference.")
+        unless $username && $reason && scalar(@args) == 0;
+
+    my $u = LJ::load_user($username);
+    return $self->error("Invalid user $username")
+        unless $u;
+
+    my $newauth = $u->generate_emailpost_auth();
+
+    return $self->print( "Token not reset" ) unless $newauth;
```

Comment:
Yes, please use $self->error, it prints differently and someone (*raises hand*) is a Bear of Very Little Brain and does not always notice that something didn't succeed unless it's printed in error styling. :)